docs/reference/configuration/site.md

```diff
@@ -149,7 +149,7 @@ Auch wenn der Modus `PV` aktiviert ist, startet die Ladung immer dann, wenn der
 Die Ladung der Hausbatterie hat unterhalb des angegebenen SoC (%) Wertes Priorität gegenüber der Fahrzeugladung.
 Steht unterhalb dieses Wertes mehr Erzeugungsleistung zur Verfügung als der Batteriespeicher aufnimmt, kann dieser Überschuss trotzdem nachrangig zur Fahrzeugladung verwendet werden.
 Wenn die Hausbatterie oberhalb des Wertes geladen wird, wird die Batterieladeleistung für das Laden des EVs als verfügbare Überschussleistung betrachtet. Somit hat dann die Fahrzeugladung Priorität bei der Verwendung der Überschussleistung.
-Ist deaktiviert (entspricht 0%) wenn kein Wert angegeben wird.
+Wird kein Wert angegeben, bleibt die Batterie unberücksichtigt (entspricht >100%).
```

Comment:
Du meinst hier 0%, oder?

Reply:
Nein. Ich habe mich nicht verschrieben. MMn muss hier für das Verständnis >100 hin.
Es ist doch nicht anderes als beim `bufferSoC`.
Oberhalb des Wertes wird die Batterie benutzt, unterhalb nicht.
Siehe https://github.com/evcc-io/evcc/issues/4652

Reply:
Also der Normalfall ist ja, dass die Batterieladung als verfügbarer Überschuss gerechnet wird.

Der prioritySoC gibt die gibt die untere Grenze dafür an. Darunter wird die Batterieladung nicht hinzuaddiert und daher kann die Batterie sich vom Überschuss nehmen was sie möchte. Die Batterieladung wird somit priorisiert.
Wenn man keine Batteriepriorisierung wünscht muss dieser Wert daher auf 0 bleiben um das Standardverhalten zu erhalten.

Reply:
Jetzt bin ich verwirrt.
ich hatte es bisher so verstanden, Wenn ich die Batterie und deren Ladung nicht nutzen möchte, dann lasse ich beide Parameter (buffer/priority) einfach weg.
Jetzt sagst du, dass in dem Fall die Batterieladung als Überschuss gewertet wird?
Dann stimmt die Doku in der bisherigen Form und ich bin von falschen Voraussetzungen ausgegangen, weil ich dachte, dass die Batterie im Normalfall immer unangetastet bleibt.
Wenn ich also die Batterieladung nicht nutzen möchte, muss ich den Wert auf > 100 einstellen?